src/registrar/templates/domain_users.html

```diff
@@ -49,7 +49,7 @@ <h1>Domain managers</h1>
   </ul>
 
   {% if domain_manager_roles %}
-  <section class="section-outlined">
+  <section class="section-outlined" id="domain-managers">
```

Comment:
If you decide to trim down on IDs, which I think is a good idea, you can test for an ID that contains the string `user-delete-form-` for the JS init